torch/_decomp/__init__.py

```diff
@@ -371,6 +371,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.unfold_backward,
             aten.unfold_copy,
             aten._unsafe_index,
+            aten._unsafe_index.Tensor,
```

Comment:
not needed. 
the entry above for `aten._unsafe_index` should already covers all the overloads.

Reply:
In that case, we don't need to make any changes for aten._unsafe_index.Tensor, since that single line is the only change in the entire diff. I'll close the PR and mark this as done.